.changeset/spicy-wombats-wash.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: `keyframe-selector-notation`/`no-unknown-animations`/`keyframe-declaration-no-important`/`keyframe-block-no-duplicate-selectors` false negatives for `@-o-keyframes` and `@-ms-keyframes`
```

Comment:
[suggest] Can you sort the rules? E.g. `keyframe-block-no-duplicate-selectors`/`keyframe-declaration-no-important`/... etc.